packages/devextreme/js/ui/card_view.d.ts

```diff
@@ -0,0 +1,1880 @@
+ */
+export type FieldCaptionClickEvent = NativeEventInfo<dxCardView, PointerEvent | MouseEvent | TouchEvent> & WithFieldCaptionInfo;
+
+/**
+ * @docid _ui_card_view_FieldCaptionDblClickEvent
+ * @public
+ * @type object
+ * @inherits NativeEventInfo,WithFieldCaptionInfo
+ */
+export type FieldCaptionDblClickEvent = NativeEventInfo<dxCardView, PointerEvent | MouseEvent | TouchEvent> & WithFieldCaptionInfo;
+
+/**
+ * @docid _ui_card_view_FieldValuePreparedEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,WithFieldValueInfo
+ */
+export type FieldValuePreparedEvent = EventInfo<dxCardView> & WithFieldValueInfo;
+
+/**
+ * @docid _ui_card_view_FieldValueClickEvent
+ * @public
+ * @type object
+ * @inherits NativeEventInfo,WithFieldValueInfo
+ */
+export type FieldValueClickEvent = NativeEventInfo<dxCardView, PointerEvent | MouseEvent | TouchEvent> & WithFieldValueInfo;
+
+/**
+ * @docid _ui_card_view_FieldValueDblClickEvent
+ * @public
+ * @type object
+ * @inherits NativeEventInfo,WithFieldValueInfo
+ */
+export type FieldValueDblClickEvent = NativeEventInfo<dxCardView, PointerEvent | MouseEvent | TouchEvent> & WithFieldValueInfo;
+
+/**
+ * @docid _ui_card_view_FieldCaptionPreparedEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,WithFieldCaptionInfo
+ */
+export type FieldCaptionPreparedEvent = EventInfo<dxCardView> & WithFieldCaptionInfo;
+
+/**
+ * @docid _ui_card_view_CardHoverChangedEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,WithCardInfo
+ */
+export type CardHoverChangedEvent = EventInfo<dxCardView> & WithCardInfo & {
+    /** @docid _ui_card_view_CardHoverChangedEvent.eventType */
+    eventType: string;
+};
+
+/**
+ * @public
+ * @docid
+ */
+export type FieldTemplateData = {
+    field: FieldInfoType;
+};
+
+/**
+ * @public
+ * @docid
+ */
+export type CardTemplateData = {
+    card: CardInfo;
+};
+
+/**
+ * @public
+ * @docid
+ */
+export type ColumnTemplateData<TCardData = unknown, TKey = unknown> = {
+    column: Column<TCardData, TKey>;
+};
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type CardCover<TCardData = unknown> = { // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     */
+    imageExpr: string | ((data: TCardData) => string);
+    /**
+     * @docid
+     * @public
+     */
+    altExpr: string | ((data: TCardData) => string);
+    /**
+     * @docid
+     * @public
+     */
+    maxHeight?: number;
+    /**
+     * @docid
+     * @public
+     */
+    aspectRatio?: string;
+    /**
+     * @docid
+     * @public
+     */
+    template?: template | ((data: CardTemplateData, container: DxElement) => string | UserDefinedElement);
+};
+
+/** @public */
+export type CardHeaderPredefinedToolbarItem = 'selectionCheckBox' | 'updateButton' | 'deleteButton';
+
+/**
+ * @docid
+ * @inherits dxToolbarItem
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type CardHeaderToolbarItem = dxToolbarItem & {
```

Comment:
```suggestion
export type CardHeaderItem = dxToolbarItem & {
```
Since it is only used in CardHeader, the `Toolbar` part is unnecessary

Reply:
Fixed.

But one thought worried me a bit
If we would like to have some items in the ```Card``` in future -> the ```CardHeaderItem``` name will be already reserved